src/app/auth/components/LogIn/LogIn.tsx

```diff
@@ -24,10 +24,17 @@ import TextInput from '../TextInput/TextInput';
 import PasswordInput from '../PasswordInput/PasswordInput';
 import { referralsThunks } from 'app/store/slices/referrals';
 import { useTranslationContext } from 'app/i18n/provider/TranslationProvider';
+import shareService from '../../../share/services/share.service';
+import notificationsService, { ToastType } from '../../../notifications/services/notifications.service';
 
 export default function LogIn(): JSX.Element {
   const { translate } = useTranslationContext();
   const urlParams = new URLSearchParams(window.location.search);
+
+  const sharingId = urlParams.get('sharingId');
```

Comment:
What happens if some param is wrong / has the wrong format / is missing?

Reply:
Nothing, it will not work